player classes description command

starspritechippy opened the pull request:
New command `$classes` to display some info regarding all classes within the actual bot itself. I don't see why there wasn't anything like this before. Not sure if there is a `classes` keyword, so I put the underscore just to be safe. 

Also, I updated command, `$hunt` a bit to actually be worth it, minimum item stat was too low. If `petlvl * 3` is too high for you, maybe we can use `petlvl * 2` instead.
I partially wanted to change this because of a recent feedback.

starspritechippy committed:
player classes description

New command `$classes` to display some info regarding all classes within the actual bot itself. I don't see why there wasn't anything like this before. Not sure if there is a `classes` keyword, so I put the underscore just to be safe. 

Also, I updated command, `$hunt` a bit to actually be worth it, minimum item stat was too low. If `petlvl * 3` is too high for you, maybe we can use `petlvl * 2` instead.
I partially wanted to change this because of a recent feedback.

starspritechippy commented:
I noticed that I accidentally used `{commands.prefix}` instead of `{ctx.prefix}`. But I'm not quite sure how to use github on mobile. If you could change that yourself if you see it, that would be perfect. Thanks!

starspritechippy committed:
Update classes.py

change `commands.prefix` to `ctx.prefix` like it's supposed to be

starspritechippy commented:
Nevermind, I found out how. Sorry about that.

Gelbpunkt commented:
If you feel like it (cuz that's what I had planned), I prefer having the `$class` command paginate through embeds giving information.
`await self.bot.paginator.ChoosePaginator(entries=[embeds], choices=["Warrior", "Mage", ...]).paginate(ctx)` would paginate them through the possibilities and interactively select one. I prefer that over another command and it looks cleaner to me

starspritechippy committed:
paginate classes

I have no experience with the paginator, actually. No idea If this would then also send the message with the embeds as well, but I hope everything is correct like this. Sorry for letting you wait.
If anything is up with the descriptions that I should change, do let me know.

Also, you didn't comment on the change to `$hunt`. Do you agree with that change?

starspritechippy commented:
I have no experience with the paginator, actually. No idea If this would then also send the message with the embeds as well, but I hope everything is correct like this. Sorry for letting you wait.
If anything is up with the descriptions that I should change, do let me know.

Also, you didn't comment on the change to `$hunt`. Do you agree with that change?

Gelbpunkt commented:
I do agree with the hunt change and will merge + fix your PR. Maybe you understand what I meant when you see it 😉

starspritechippy commented:
Ah, I think I understand. Maybe.
Perhaps there will be more paginating to do in the future and I could give it another go. :smile:
